Update code hotspots documentation text (#17801)
DataDog/documentation: content/en/profiler/connect_traces_and_profiles.md

### Link from a span to profiling data

From the view of each trace, the Code Hotspots tab highlights profiling data scoped on the selected spans.

The values on the left side is the time spent in that method call during the selected span. Depending on the runtime and language, this list of types varies:

- **Method durations** shows the overall time taken by each method from your code.
- **CPU** shows the time taken executing CPU tasks.
- **Synchronization** shows the time spent waiting on monitors, the time a thread is sleeping and the time it is parked.
- **VM operations** (Java only) shows the time taken waiting for VM operations that are not related to garbage collection (for example, heap dumps).
- **File I/O** shows the time taken waiting for a disk read/write operation to execute.
- **Socket I/O** shows the time taken waiting for a network read/write operation to execute.
- **Monitor enter** shows the time a thread is blocked on a lock.
- **Uncategorized** shows the time taken to execute the span that cannot be placed into one of the above categories.

> Clicking on the plus `+` expands the stack trace to that method **in reverse order**. Hovering over the value shows what percentage of time is explained by which category.

Reviewer: Would "expand the the stack trace **of** that method" change the meaning here?

```suggestion
Click the plus icon `+` to expand the stack trace to that method **in reverse order**. Hover over the value to see the percentage of time explained by category.
```

### Viewing a profile from a trace

{{< img src="profiler/flamegraph_view.mp4" alt="Opening a view of the profile in a flame graph" video=true >}}

> For each type from the breakdown, click **View In Full Page** to view the same data in the full page view. From there you can change visualization to the flame graph.

Reviewer: Trying to reword it so that we don't keep repeating view full page, but might not be as effective. 

```suggestion
For each type from the breakdown, click **View In Full Page** to see the same data opened up in a in a new page . From there you can change visualization to the flame graph.
```

Click the **Focus On** selector to define the scope of the data:

> - **Span & Children** scopes the profiling data to the selected span and all descendant spans in the same service.

Reviewer: Is this listed with **&** or spelled out? If this matches the UI, this is fine.

Author: Yes, this is exactly as it's in the UI.
